common/presenters/message-banner/move-to-confirmed-assessment-banner.js

```diff
@@ -0,0 +1,35 @@
+const i18n = require('../../../config/i18n')
+const filters = require('../../../config/nunjucks/filters')
+
+module.exports = function ({ move, baseUrl, context }) {
+  const assessment = move.profile[context]
+  const content = `
+    <p>
+      ${i18n.t(`messages::assessment.${assessment.status}.content`, {
+        context,
+        date: filters.formatDateWithTimeAndDay(assessment.confirmed_at),
+      })}
+    </p>
+
+    <p>
+      <a href="${baseUrl}/print" class="app-icon app-icon--print">
+        ${i18n.t('actions::print_assessment', {
+          context,
+        })}
+      </a>
+    </p>
+  `
+
+  return {
```

Comment:
These blocks do feel slightly duplicated. I was toying with the idea of move the consisted properties back out to the parent method and maybe each of these just returns title/content?

But then I don't know in what sort of format it should return.

a flatter structure: 
```javascript
return {
  title: 'title',
  content: '<p>content</p>'
}

const presenter = bannerPresenter({ ... })
return {
  allowDismiss: false,
  classes: 'app-message--instruction govuk-!-padding-right-0',
  title: {
    text: presenter.title,
  },
  content: {
    html: presenter.content,
  },
}
```

or the same structure as the component: 
```javascript
return {
  title: {
    text: 'title',
  },
  content: {
    html: '<p>content</p>',
  }
}

return {
  allowDismiss: false,
  classes: 'app-message--instruction govuk-!-padding-right-0',
  ...presenter,
}
```